### PR TITLE
[Polkadot] prepare for MultiAddress::AccountId

### LINKED ISSUE #1303: [Polkadot] can't estimate fee or send tx spec 28
sidecar errors

```
"1: Unable to query dispatch info.: Execution, ApiCould not convert parameter `uxt` between node and runtime: No such variant in enum MultiAddress",
```

```
createType(ExtrinsicV4):: createType(ExtrinsicSignatureV4):: Unable to create Enum via index 157, in Id, Index, Raw, Address32, Address20
```

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #1303

### CODE REVIEW

File: src/Polkadot/Extrinsic.h [hunk 1]
```diff
@@ -41,8 +43,10 @@ class Extrinsic {
         if (input.has_era()) {
             era = encodeEra(input.era().block_number(), input.era().period());
         } else {
-            era = encodeCompact(0);
+          // immoral era
```
Comment (catenocrypt) [inline src/Polkadot/Extrinsic.h:46]: 'immortal' instead of 'immoral'. :D , or just 'era'

**Review by catenocrypt — APPROVED** [review]
Are there sufficient tests for both spec >= 28 and < 28 cases?  Otherwise LGTM

File: src/Polkadot/Extrinsic.h (hunk 1 above)
Comment (hewigovens) [reply src/Polkadot/Extrinsic.h:46]: 😂 will fix this typo

**Comment by hewigovens** [comment]
@catenocrypt old tests were created before (less than spec 28), I just added a new transfer test, will add staking tests later (let's release asap)

**Review by vikmeup — APPROVED** [review]

**Pull request merged**